src/openfluid/tools/StringHelpers.cpp

```diff
@@ -173,6 +174,24 @@ std::string join(const std::vector<std::string>& Vect, const std::string& Sep)
 }
 
 
+// =====================================================================
+// =====================================================================
+
+
+bool match(const std::string& Str1, const std::string& Str2)
```

Comment:
Make it asymetrical: we only want an element compared to a reference to avoid any ambiguity in the check